CHANGELOG.md

```diff
@@ -2,6 +2,85 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [2.0.0](https://github.com/ably/ably-js/tree/2.0.0) (date TBD)
+
+The 2.0.0 release introduces a number of significant changes. Here is an overview:
+
+- Support for Internet Explorer has been dropped.
+- Support for Node.js versions lower than 16 has been dropped. The supported Node.js versions are now 16, 18, and 20.
+- The minimum supported versions for major browsers are: Chrome 58, Firefox 52, Edge 79, Safari 11, and Opera 45.
+- An ESM variant of the library is now available for browsers (but not for Node.js) at `ably/modules`. This modular variant of the library supports tree shaking, allowing for a reduction in the Ably bundle size within your application.
+- React hooks, exported at `ably/react`, now rely on the `ChannelProvider` component to define the channels you wish to use and the options for them.
+- The exports `ably/callbacks` and `ably/promises` have been removed. The callbacks API has been entirely removed, and now only promises are supported. Calls like `new Ably.Realtime.Promise` should be replaced with `new Ably.Realtime`.
+- The browser bundle now relies on the native Web Crypto API instead of CryptoJS. The `ably/build/ably.noencryption` bundle has been removed, as it is no longer necessary.
+- The separate bundle for web workers, `ably/build/ably-webworker`, has been removed. You can now import from `ably` in web workers.
+- The Types namespace has been removed. All types are now exported at the top level.
+
+Read more in [the migration guide for v2](link TBD).
+
+BREAKING CHANGES
+
+- Remove code that's supporting older platforms [\#1629](https://github.com/ably/ably-js/pull/1629)
+- `ChannelProvider` implementation [\#1620](https://github.com/ably/ably-js/pull/1620)
+- Remove duplicate type names and change public `ChannelModes` type [\#1601](https://github.com/ably/ably-js/pull/1601)
+- Remove `any` from `stats()` param type [\#1561](https://github.com/ably/ably-js/pull/1561)
+- Make modular variant of the library available to Web Workers [\#1550](https://github.com/ably/ably-js/pull/1550)
```

Comment:
This title doesn't really describe a breaking change - the real breaking change is that we've removed the web-worker specific build

Reply:
Agree, will change

Reply:
see https://github.com/ably/ably-js/pull/1656/commits/dd8db9409cb3cde90bfc321ddd33ea71bd3009c2